### DISCUSSION TITLE
Viewer List Update

### DISCUSSION THREAD

**Discussion by yomiyo17** [KevinBytesTheDust/CVAmp#161] [General]
Just a curious question but is there currently any restriction that is stopping the Proxies/Instances from showing up in the viewer list as actual names, also does this have something to do with creating actual twitch accounts for the bots and would that have to be done by the end user?

**Comment by jlplenio** [comment]
Instances increase the viewer count. If you want the instances to show up as users in the chat user list, then you need to have [auth_tokens for each of your twitch accounts](https://twitch-dl.bezdomni.net/commands/download.html#downloading-subscriber-only-vods). This way the instances can log in. I released a preview, where you can add the tokens to CTVBot and the viewers will show in the chat user list, here: https://github.com/jlplenio/crude-twitch-viewer-bot/discussions/163

**Reply by yomiyo17** [reply]
Wow thank you so much for replying, By any chance is this out yet/soon and if so how do I go about adding tokens.
